packages/repository-json-schema/src/build-schema.ts

```diff
@@ -16,10 +16,44 @@ import {JSONSchema6 as JSONSchema} from 'json-schema';
 import {JSON_SCHEMA_KEY, MODEL_TYPE_KEYS} from './keys';
 
 export interface JsonSchemaOptions {
+  /**
+   * Set this flag if you want the schema to define navigational properties
+   * for model relations.
+   */
   includeRelations?: boolean;
+
+  /**
+   * Set this flag to mark all model properties as optional. This is typically
+   * used to describe request body of PATCH endpoints.
+   */
+  partial?: boolean;
+
+  /**
+   * @private
+   */
   visited?: {[key: string]: JSONSchema};
 }
 
+/**
+ * @private
+ */
+export function buildModelCacheKey(
+  ctor: Function,
```

Comment:
Is there a reason for the `ctor` parameter in this function? I don't see it being used.

Reply:
Good catch, I'll remove it.